src/Objs/Event/EventEditingConfig.js

```diff
@@ -68,6 +68,33 @@ Scrivito.provideEditingConfig("Event", {
   propertiesGroups: [socialCardsPropertiesGroup, metadataPropertiesGroup],
   initialContent: {
     ...metadataInitialContent,
+    title: "Lorem Ipsum",
     body: [new SectionWidget({})],
   },
+  validations: [
+    [
+      "title",
+
+      title => {
+        if (!title) {
+          return {
+            message: "The event title must be set.",
+            severity: "error",
+          };
+        }
+      },
+    ],
+    [
+      "date",
+
+      date => {
+        if (!date) {
+          return {
+            message: "The event date is not set.",
```

Comment:
Let's wait until https://github.com/Scrivito/scrivito_example_app_js/pull/266#discussion_r360318495 is decided.